postgres/datadog_checks/postgres/config.py

```diff
@@ -64,8 +64,9 @@ def __init__(self, instance):
         self.service_check_tags = self._get_service_check_tags()
         self.custom_metrics = self._get_custom_metrics(instance.get('custom_metrics', []))
         self.max_relations = int(instance.get('max_relations', 300))
-        self.max_query_metrics = int(instance.get('max_query_metrics', 300))
+        self.max_query_metrics = int(instance.get('max_query_metrics', 100000))
```

Comment:
Let's just remove it (see comment below)

Reply:
Removed